docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -15,7 +15,7 @@ The namespace in which to deploy this component.
 
 [horizontal]
 type:: string
-default:: `v1.7.4`
+default:: `v2.2.2`
```

Comment:
I don't think it's worth maintaining the version here also. (Especially if there might be Renovate support in the future)
```suggestion
default:: see `defaults.yaml`
```